core/feed.py

```diff
@@ -20,7 +20,7 @@ def render_feed(feed_id: str, plugin: Plugin, options: GlobalOptions, base_url:
                 id=episode.item_id,
                 title=episode.title,
                 media=Media(generate_url(episode, plugin, options, base_url), episode.content_length, type=episode.content_type),
-                summary=episode.description.replace("\n", "<br>") if episode.description and options.html_newlines else episode.description,
```

Comment:
Remove this workaround. Now it is the responsibility of the plugin to output html in the summary field. This way we can extract directly html from the original feed.